lib/utils/isAutoprefixable.js

```diff
@@ -1,22 +1,273 @@
+	'columns',
+	'filter',
+	'flex',
+	'flex-basis',
+	'flex-direction',
+	'flex-flow',
+	'flex-grow',
+	'flex-shrink',
+	'flex-wrap',
+	'flow-from',
+	'flow-into',
+	'font-feature-settings',
+	'font-kerning',
+	'font-language-override',
+	'font-variant-ligatures',
+	'grid-area',
+	'grid-column',
+	'grid-column-align',
+	'grid-column-end',
+	'grid-column-start',
+	'grid-row',
+	'grid-row-align',
+	'grid-row-end',
+	'grid-row-start',
+	'grid-template',
+	'grid-template-areas',
+	'grid-template-columns',
+	'grid-template-rows',
+	'hyphens',
+	'image-rendering',
+	'justify-content',
+	'margin-block-end',
+	'margin-block-start',
+	'margin-inline-end',
+	'margin-inline-start',
+	'mask',
+	'mask-border',
+	'mask-border-outset',
+	'mask-border-repeat',
+	'mask-border-slice',
+	'mask-border-source',
+	'mask-border-width',
+	'mask-clip',
+	'mask-composite',
+	'mask-image',
+	'mask-origin',
+	'mask-position',
+	'mask-repeat',
+	'mask-size',
+	'object-fit',
+	'object-position',
+	'order',
+	'overscroll-behavior',
+	'padding-block-end',
+	'padding-block-start',
+	'padding-inline-end',
+	'padding-inline-start',
+	'perspective',
+	'perspective-origin',
+	'place-self',
+	'region-fragment',
+	'scroll-snap-coordinate',
+	'scroll-snap-destination',
+	'scroll-snap-points-x',
+	'scroll-snap-points-y',
+	'scroll-snap-type',
+	'shape-image-threshold',
+	'shape-margin',
+	'shape-outside',
+	'tab-size',
+	'text-align-last',
+	'text-decoration',
+	'text-decoration-color',
+	'text-decoration-line',
+	'text-decoration-skip',
+	'text-decoration-skip-ink',
+	'text-decoration-style',
+	'text-emphasis',
+	'text-emphasis-color',
+	'text-emphasis-position',
+	'text-emphasis-style',
+	'text-orientation',
+	'text-overflow',
+	'text-size-adjust',
+	'text-spacing',
+	'touch-action',
+	'transform',
+	'transform-origin',
+	'transform-style',
+	'transition',
+	'transition-delay',
+	'transition-duration',
+	'transition-property',
+	'transition-timing-function',
+	'user-select',
+	'writing-mode',
+]);
+
+/**
+ * @example
+ * Object.values(prefixes.remove)
+ *   .filter((p) => Array.isArray(p.values))
+ *   .flatMap((p) => p.values)
+ *   .map((p) => p.prefixed);
+ */
+const PROPERTY_VALUES = new Set([
+	'-moz-available',
+	'-moz-box',
+	'-moz-calc',
+	'-moz-crisp-edges',
+	'-moz-element',
+	'-moz-fit-content',
+	'-moz-grab',
+	'-moz-grabbing',
+	'-moz-inline-box',
+	'-moz-isolate',
+	'-moz-isolate-override',
+	'-moz-linear-gradient',
+	'-moz-max-content',
+	'-moz-min-content',
+	'-moz-plaintext',
+	'-moz-radial-gradient',
+	'-moz-repeating-linear-gradient',
+	'-moz-repeating-radial-gradient',
+	'-moz-zoom-in',
+	'-moz-zoom-out',
+	'-ms-flexbox',
+	'-ms-grid',
+	'-ms-inline-flexbox',
+	'-ms-inline-grid',
+	'-ms-linear-gradient',
+	'-ms-radial-gradient',
+	'-ms-repeating-linear-gradient',
+	'-ms-repeating-radial-gradient',
+	'-o-linear-gradient',
+	'-o-pixelated',
+	'-o-radial-gradient',
+	'-o-repeating-linear-gradient',
+	'-o-repeating-radial-gradient',
+	'-webkit- oldlinear-gradient',
```

Comment:
Are the property values with space inside OK?

Reply:
@victor-homyakov This list is output by Autoprefixer. Here is the snippet to produce the list:

`package.json`:

```json
{
  "dependencies": {
    "autoprefixer": "10.2.5"
  }
}
```

`index.js`:

```js
const autoprefixer = require("autoprefixer");
const Browsers = require("autoprefixer/lib/browsers");
const Prefixes = require("autoprefixer/lib/prefixes");
const prefixes = new Prefixes(
  autoprefixer.data.prefixes,
  new Browsers(autoprefixer.data.browsers, [])
);

const list = Object.values(prefixes.remove)
  .filter((p) => Array.isArray(p.values))
  .flatMap((p) => p.values)
  .map((p) => p.prefixed);
console.log(new Set(list.sort()));
```

Run:

```console
$ docker run --rm -v $PWD:/work -w /work node:16.2.0 sh -c 'npm install && node index.js'

up to date, audited 14 packages in 2s

4 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Set(60) {
  '-moz-available',
  '-moz-box',
  '-moz-calc',
  '-moz-crisp-edges',
  '-moz-element',
  '-moz-fit-content',
  '-moz-grab',
  '-moz-grabbing',
  '-moz-inline-box',
  '-moz-isolate',
  '-moz-isolate-override',
  '-moz-linear-gradient',
  '-moz-max-content',
  '-moz-min-content',
  '-moz-plaintext',
  '-moz-radial-gradient',
  '-moz-repeating-linear-gradient',
  '-moz-repeating-radial-gradient',
  '-moz-zoom-in',
  '-moz-zoom-out',
  '-ms-flexbox',
  '-ms-grid',
  '-ms-inline-flexbox',
  '-ms-inline-grid',
  '-ms-linear-gradient',
  '-ms-radial-gradient',
  '-ms-repeating-linear-gradient',
  '-ms-repeating-radial-gradient',
  '-o-linear-gradient',
  '-o-pixelated',
  '-o-radial-gradient',
  '-o-repeating-linear-gradient',
  '-o-repeating-radial-gradient',
  '-webkit- oldlinear-gradient',
  '-webkit- oldradial-gradient',
  '-webkit- oldrepeating-linear-gradient',
  '-webkit- oldrepeating-radial-gradient',
  '-webkit-box',
  '-webkit-calc',
  '-webkit-cross-fade',
  '-webkit-fill-available',
  '-webkit-filter',
  '-webkit-fit-content',
  '-webkit-flex',
  '-webkit-grab',
  '-webkit-grabbing',
  '-webkit-image-set',
  '-webkit-inline-box',
  '-webkit-inline-flex',
  '-webkit-isolate',
  '-webkit-linear-gradient',
  '-webkit-max-content',
  '-webkit-min-content',
  '-webkit-optimize-contrast',
  '-webkit-radial-gradient',
  '-webkit-repeating-linear-gradient',
  '-webkit-repeating-radial-gradient',
  '-webkit-sticky',
  '-webkit-zoom-in',
  '-webkit-zoom-out'
}
```

I don't know why Autoprefixer outputs a whitespace-included property such as `-webkit- oldradial-gradient`.
But I think this will no be a problem in most cases because such properties are outdated.

Do you have some idea to handle them?

Reply:
I looked into the code of autoprefixer.

It marks with `' old'` cases with more than one possible syntax, e.g. `'-webkit- old'`, `'-ms- old'`, at least [here](https://github.com/postcss/autoprefixer/blob/10.2.5/data/prefixes.js#L191) and [here](https://github.com/postcss/autoprefixer/blob/10.2.5/lib/hacks/placeholder.js#L8).

Then such prefixes are handled in a special way
- [here](https://github.com/postcss/autoprefixer/blob/10.2.5/lib/utils.js#L24) the utility function removes `' old'` part from prefix
- [here](https://github.com/postcss/autoprefixer/blob/10.2.5/lib/hacks/gradient.js#L20) and [here](https://github.com/postcss/autoprefixer/blob/10.2.5/lib/hacks/placeholder.js#L21-L26) are separate code paths for such prefixes

I think that for purposes of this PR you can use [utility function](https://github.com/postcss/autoprefixer/blob/10.2.5/lib/utils.js#L24), which removes `' old'` from vendor prefix, i.e. you'll end up with `'-webkit-linear-gradient'` (valid) instead of `'-webkit- oldlinear-gradient'` (invalid).

Reply:
@victor-homyakov Thank you so much for the investigation! 👏🏼 

Are you interested in creating a new PR to fix the invalid properties?
(If no, I'll create it instead.)

Reply:
I'd like to create a PR, but I'm very busy next two weeks

Reply:
@victor-homyakov No problems. I've created PR #5321.
Thank you so much for your investigation! 😊